package.json

```diff
@@ -9,12 +9,14 @@
   },
   "scripts": {
     "build": "webpack --mode production",
-    "start": "npm run build && node ./src/server/bin/server",
+    "start": "npm run db:migrate && npm run build && node ./src/server/bin/server",
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
     "server": "nodemon ./src/server/bin/server",
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "db:clean": "cd ./src/server/ && knex migrate:rollback",
-    "db:setup": "cd ./src/server/ && knex migrate:latest && knex seed:run",
+    "db:setup": "npm run db:migrate && npm run db:seed",
+    "db:migrate": "cd ./src/server/ && knex migrate:latest",
+    "db:seed": "cd ./src/server/ && knex seed:run",
     "db:reset": "npm run db:clean && npm run db:setup",
     "storybook": "cd ./src/client/ && start-storybook -p 3007",
```

Comment:
@sofiiadidovych Regarding setup database in Herku the following script should be added as well.`"heroku-postbuild": "npm run db:setup"`

Reply:
@arya56 Thank you for your review. 
I added "heroku-postbuild" script. As far as I understand, we should only run migration script without seeding. The problem with the seeding is that it runs all the seed files every time, meaning we will delete all the existing data and insert the new data after every deployment.

Reply:
@dpfernandes thank you for your clarification on this issue. @arya56 I changed my "heroku-postbuilt" command as you asked: "heroku-postbuild": "npm run db:setup"
Hope this PR is ready for merging :)